calendar/calendar-tool/tool/src/webapp/js/sakai-calendar-fields.js

```diff
@@ -0,0 +1,65 @@
+const confModal = document.getElementById('confirmation-modal');
```

Comment:
where is this used?

Reply:
Good catch. I must have added this due to this id's many references in the jquery embedded in the velocity template (pre-23.x); however, its explicit reference in JS is no longer needed now that we're using bootstrap modals.